src/storage/backend/redis/Redis.h

```diff
@@ -15,18 +15,18 @@ struct redisPollEvents;
 namespace zeek::storage::backend::redis {
 class Redis : public Backend, public iosource::IOSource {
 public:
-    Redis(std::string_view tag) : Backend(SupportedModes::ASYNC, tag), IOSource(true) {}
+    Redis() : Backend(SupportedModes::ASYNC, "REDIS"), IOSource(true) {}
     ~Redis() override = default;
 
-    static BackendPtr Instantiate(std::string_view tag);
+    static BackendPtr Instantiate();
 
     /**
      * Returns a descriptive tag representing the source for debugging.
      * This has to be overloaded for Redis because IOSource requires it.
      *
      * @return The debugging name.
      */
-    const char* Tag() override { return tag.c_str(); }
+    const char* Tag() override { return tag_str.c_str(); }
```

Comment:
I noticed that if you declare `Tag()` virtual in Backend.h, you can skip it here. That seems nice since it's the same implementation, and then one less place where you need `tag_str`.